### PR TITLE
Initial redbiom recipe

### PR DESCRIPTION
**Pull request by wasade**
Checklist

- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vend other packages
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/redbiom/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/biocore/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: b5a1f8a8e6d81159fbe639cffd2aff0abffc769247997ff19f1f36755c1f9da7
+
+build:
+  skip: true  # [win or py2k]
```
Comment (xhochy) [inline recipes/redbiom/meta.yaml]: Could this package be `noarch`? Doesn't seem to have compiled code.

File: recipes/redbiom/build.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+export LC_ALL=en_US.utf8
+export LANG=en_US.utf8
+$PYTHON setup.py install --single-version-externally-managed --record=record.txt
```
Comment (xhochy) [inline recipes/redbiom/build.sh]: Can you use `pip`?

File: recipes/redbiom/meta.yaml (hunk 1 above)
Comment (wasade) [reply recipes/redbiom/meta.yaml]: `biom-format` and `scikit-bio` both do and don't support windows. I did relax this constraint though as I don't *think* it needs to be explicitly enforced here.

File: recipes/redbiom/build.sh (hunk 2 above)
Comment (wasade) [reply recipes/redbiom/build.sh]: Done!

File: recipes/redbiom/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/biocore/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: b5a1f8a8e6d81159fbe639cffd2aff0abffc769247997ff19f1f36755c1f9da7
+
+build:
+  skip: true
```
Comment (xhochy) [inline recipes/redbiom/meta.yaml]: Through this, you aren't building anything at the moment, package can be noarch, so better replace this with:
```suggestion
  noarch: python
```

File: recipes/redbiom/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+build:
+  skip: true  
+  number: 0
+  entry_points:
+    - redbiom=redbiom.commands:cli
+
+requirements:
+  host:
+    - python
```
Comment (xhochy) [inline recipes/redbiom/meta.yaml]: Align host and run:
```suggestion
    - python >=3.5
```

**Comment by xhochy** [comment]
Failure on Windows is ok as this is a noarch package.

**Comment by wasade** [comment]
Thanks, @xhochy!

**Pull request merged**